examples/platforms/da15000/radio.c

```diff
@@ -95,15 +97,17 @@ void da15000RadioInit(void)
 void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
 {
     (void)aInstance;
+    uint32_t *factoryTestTimeStamp  = (uint32_t *) FACTORY_TEST_TIMESTAMP;
+    uint32_t *factoryTestId         = factoryTestTimeStamp + 1;
```

Comment:
I don't understand. Just basic pointer operations. How to make it more explicit?